DEVELOPMENT.md

```diff
@@ -35,45 +35,54 @@ Install these tools:
     ```bash
     kind create cluster
     ```
+   **NOTE**: The KinD cluster can also be created with tools like 
+   [Podman Desktop](https://podman-desktop.io/docs/kind/creating-a-kind-cluster).
+ 
 
 1. Set up `ko` to upload images to the KinD cluster, or any other registry you
   want to use:
     ```bash
     export KO_DOCKER_REPO="kind.local"
     ```
+   **NOTE**: if your cluster doesn't have the name `kind` you need to specify the env var `KIND_CLUSTER_NAME`.
```

Comment:
I don't think this is needed. People that want to go further and make an advanced usage of tools can do it by their own.

Reply:
I had to look for this when I started to use `ko` and not knowing this made face issues. I'm open to move it elsewhere, maybe to the "Known issues" section, but I feel this should be written somewhere.